README.md

```diff
@@ -63,11 +63,11 @@ Congratulations, you have just built your first Haystack app!
 
 ## Core Concepts
 
-🏃‍♀️ **[Pipelines](https://docs.haystack.deepset.ai/docs/pipelines):** This is the standard Haystack structure that builds on top of your data to perform various NLP tasks such as retrieval augmented generation, question answering and more. The data in a Pipeline flows from one Node to the next. You define how Nodes interact with each other, and how one Node pushes data to the next.
+🏃‍♀️ **[Pipelines](https://docs.haystack.deepset.ai/docs/pipelines):** This is the standard Haystack structure that builds on top of your data to perform various NLP tasks such as retrieval augmented generation, question answering and more. The data in a Pipeline flows from one Node to the next. You define how Nodes interact with each other and how one Node pushes data to the next.
 
-An example pipeline would consist of one `Retriever` Node and one `PromptNode`. When the pipeline runs with a query, the Retriever first retrieves the relevant context to the query from your data, and then the PromptNode uses an LLM to generate the final answer.
+An example pipeline would consist of one `Retriever` Node, and one `PromptNode`. When the pipeline runs with a query, the Retriever first retrieves the relevant context to the query from your data and then the PromptNode uses an LLM to generate the final answer.
```

Comment:
```suggestion
An example pipeline would consist of one `Retriever` Node and one `PromptNode`. When the pipeline runs with a query, the Retriever first retrieves the relevant context to the query from your data, and then the PromptNode uses an LLM to generate the final answer.
```

There are only two items in the first sentence, so Oxford comma does not work in this case. In the second sentence, the comma is needed as there are two subjects and predicates, so they work like two separate sentences.